browser/js/scroll/scroll.directive.js

```diff
@@ -76,14 +73,20 @@ core.directive('blLetterScroll', function(ScrollFactory, KeyboardFactory, Positi
             function drawLoop() {
                 requestAnimationFrame(drawLoop);
                 context.clearRect(0, 0, canvas.width, canvas.height);
-                ctracker.draw(canvas);
+                scope.ctracker.draw(canvas);
             }
 
+            var errorCallback = function(e) {
```

Comment:
Let's create an error factory to put all error handling and alert messages and stuff.